c++/src/kj/async-io-win32.c++

```diff
@@ -1086,8 +1086,9 @@ public:
       : lowLevel(parent.lowLevel), filter(allow, deny, parent.filter) {}
 
   Promise<Own<NetworkAddress>> parseAddress(StringPtr addr, uint portHint = 0) override {
-    return evalLater([this,portHint,addr=kj::mv(addr)]() {
-      return SocketAddress::parse(lowLevel, addr, portHint, filter);
+    auto addrCopy = kj::str(addr);
+    return evalLater([this,portHint,addrCopy=kj::mv(addrCopy)]() {
+      return SocketAddress::parse(lowLevel, addrCopy, portHint, filter);
```

Comment:
Simpler resolution:

```suggestion
    return evalLater([this,portHint,addr=kj::str(addr)]() {
      return SocketAddress::parse(lowLevel, addr, portHint, filter);
```

Reply:
Hmm, though maybe this would be more efficient?

```suggestion
    return evalNow([&]() {
      return SocketAddress::parse(lowLevel, addr, portHint, filter);
```

Reply:
This seems like an improvement. I can't figure out why it was `evalLater` to begin with. I'm also surprised that this hasn't caused issues on Unix more often. In theory you could say it's the caller's fault for not attaching the string to the promise but it seems like I've commonly forgot to do that myself.